src/client/views/start.ts

```diff
@@ -10,7 +10,7 @@ export class Start extends AbstractView {
               <div class="brand-tn"></div>
               <div class="inner-content-tn">
                 <div class="inner-content-block-tn">
-                  <button class="opening-btn-tn">Let's go!</button>
+                  <button class="opening-btn-tn" aria-label="Start negotiator button">Let's go!</button>
```

Comment:
How about we use 'click to begin connecting your tokens'

Reply:
Token Negotiator or Brand Connector is not ideal because we use these names interchangeably.